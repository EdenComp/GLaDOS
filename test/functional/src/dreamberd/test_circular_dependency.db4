import "../samples/basic.db4";

print("I'm the last one here!\n");
