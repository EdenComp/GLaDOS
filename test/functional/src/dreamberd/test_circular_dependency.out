Bien le bonjour
Je suis assez relou!
I'm the last one here!
